aws/resource_aws_codebuild_project.go

```diff
@@ -213,6 +238,11 @@ func resourceAwsCodeBuildProjectCreate(d *schema.ResourceData, meta interface{})
 		params.TimeoutInMinutes = aws.Int64(int64(v.(int)))
 	}
 
+	if _, ok := d.GetOk("vpc_config"); ok {
```

Comment:
I would change this to (note about function signature update below in another comment):
```
if v, ok := d.GetOk("vpc_config"); ok {
	params.VpcConfig = expandCodeBuildVpcConfig(v.([]interface{})[0].(map[string]interface{}))
}
```